src/isomorphic/classic/types/ReactPropTypes.js

```diff
@@ -249,7 +255,13 @@ function createObjectOfTypeChecker(typeChecker) {
 }
 
 function createUnionTypeChecker(arrayOfTypeCheckers) {
-  function validate(props, propName, componentName, location, propFullName) {
+  if (!Array.isArray(arrayOfTypeCheckers)) {
+    return new Error(
+      `Invalid argument supplied to oneOfType, expected an instance of array.`
+    );
+  }
+
+  function validate(props, propName, componentName, location) {
```

Comment:
The lint error is because you inadvertently removed a param here (`propFullName`)